datasets/ade_corpus_v2/README.md

```diff
@@ -0,0 +1,219 @@
+[Needs More Information]
```

Comment:
Could you add the tags as YAML here ? (not JSON, even though the tagger allows to save as JSON) ?
You can get the YAML tags at the upper right corner of the tagger app, by clicking on `expand yaml tags`

You can find an example of what you should get by using the tagger app for the dataset card [here](https://raw.githubusercontent.com/huggingface/datasets/ce646560d7166b2d2e519625583cf2d523063838/datasets/has_part/README.md)

You can remove the `tags.json` files